app/models/concerns/user_capabilities.rb

```diff
@@ -6,12 +6,18 @@ module UserCapabilities
     attribute :metadata, ActiveRecord::Type::Json.new
   end
 
+  class_methods do
+    def policy_class
+      @policy_class ||= "#{self}Policy".constantize
```

Comment:
Should this be a class variable instead? `@@policy_class`? I mean, all instances will have the same `policy_class`, but I don't know what the implications are of setting an instance variable in a class method.

Reply:
@eclarizio It is a class method and hence a class variable, unlike the @@

Reply:
As @mkanoor pointed out, even though it only has one `@`, it still is a class variable, because declaring `class Something`, you are actually creating an instance of a `Class` object, so when you add a class method with `@variable`, that is a class variable for the `Something` class but an instance variable for the `Class` object. Pretty interesting and I forgot that was how Ruby handled classes.